### ISSUE TITLE
measure support for URL.toObjectURL ??

### ISSUE THREAD

**Issue by jokeyrhyme** [WebPlatformTest/HTML5test#324]
Just a proposal.

There's a Web API that facilitates storing arbitrary Blobs and referring to them elsewhere as though they are ordinary resources:
https://developer.mozilla.org/en-US/docs/Web/API/URL.createObjectURL

One use case might be storing images in IndexedDB, retrieving them as Blobs, creating URLs for these Blobs, and using those URLs in `img` elements in the page.

Is it worth highlighting support (or lack thereof) for this?

**Comment by jokeyrhyme** [comment]
This is technically part of the File API, so you may already have it covered.

**Comment by NielsLeenheer** [comment]
Is there any reason you believe that this should be test separately? Do you know of any browser that does support the main part of the File API, but not toObjectURL?

**Comment by jokeyrhyme** [comment]
http://caniuse.com/#feat=bloburls
http://caniuse.com/#feat=filereader

@NielsLeenheer according to the above: Android 3, IEMobile 10, Opera\* 11.1 to 12.1 all implement FileReader but don't implement Blob URLs.

There are currently 10 points for the Files section. What if we made one of those points contingent on Blob URLs?

**Comment by NielsLeenheer** [comment]
OK, that is enough reason to split this up. I'll look into it for the next release of HTML5test.

**Comment by NielsLeenheer** [comment]
Added separate tests for various aspects of the File API: FileReader, Blob, readAsArrayBuffer, readAsDataURL and createObjectURL

**Comment by jokeyrhyme** [comment]
This is commit: b4bad9afafcbdffa56a8dcae6c53e3b9efebcc8a
Just FYI.
